test/k8sT/Services.go

```diff
@@ -1668,11 +1676,19 @@ var _ = Describe("K8sServicesTest", func() {
 						var ccnpHostPolicy string
 
 						BeforeAll(func() {
-							DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
-								"global.tunnel":               "disabled",
-								"global.autoDirectNodeRoutes": "true",
-								"global.hostFirewall":         "true",
-							})
+							options := map[string]string{
+								"tunnel":               "disabled",
+								"autoDirectNodeRoutes": "true",
+								"hostFirewall":         "true",
```

Comment:
I think these need `global.` since we're before v1.9.

Reply:
How did CI pass? :exploding_head:

Reply:
I think CI passed because the host policy was simply not applied and we don't check that some subset of traffic is dropped. I think we only check that some subset is forwarded. So in allow-all mode, everything is forwarded and the test passes. I guess it speaks to the fragility of this test :grimacing: